### PR TITLE
feat: add keda autoscaling support

### PR DESCRIPTION
**Pull request by Ian2012**
### Description

This PR allows to add autoscaling values for the celery deployment variants.

### CODE REVIEW

File: tutorcelery/templates/celery/k8s/keda.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      address: redis.{{K8S_NAMESPACE}}:{{REDIS_PORT}}
+      {% else %}
+      address: {{REDIS_HOST}}:{{REDIS_PORT}}
+      {% endif %}
+      listLength: "{{ config.get("list_length", 40)}}"
+      listName: edx.{{service}}.core.{{variant}}
+    type: redis
+{% endfor %}
+{% endfor %}
+{% endif %}
```
Comment (jfavellar90) [inline tutorcelery/templates/celery/k8s/keda.yml]: Line at the end of the file

File: tutorcelery/templates/celery/k8s/keda.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{% if CELERY_MULTIQUEUE_ENABLED and CELERY_ENABLE_KEDA_AUTOSCALING %}
```
Comment (jfavellar90) [inline tutorcelery/templates/celery/k8s/keda.yml]: This makes me think that `CELERY_ENABLE_KEDA_AUTOSCALING` is only valid for `CELERY_MULTIQUEUE_ENABLED=true`. You won't be able to configure Keda if you have just the regular worker right?

Comment (Ian2012) [reply tutorcelery/templates/celery/k8s/keda.yml]: Yes, you are right.

File: tutorcelery/templates/celery/k8s/keda.yml (hunk 1 above)
Comment (jfavellar90) [reply tutorcelery/templates/celery/k8s/keda.yml]: amm it is still present according to Github

**Review by Henrrypg — APPROVED** [review]
LGTM

**Review by jfavellar90 — APPROVED** [review]

**Pull request merged**